docker/bin/certify.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+DOCKER_BIN="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+export PASS="abcdefgh"
+source ${DOCKER_BIN}/../.env
+
+if [ -f ${TLS}/ca-cert ]; then
+    echo "${TLS}/ca-cert found; skipping certificate generation.."
+    exit 0
+fi
+
+# Clean up old certs
+#for file in $(ls ${TLS});do
```

Comment:
`for file in $TLS/*` ?

Reply:
Primed for parameter expansion. This is inherently unsafe anyway I really should just remove it and add a comment about cleaning up old certs